### ISSUE TITLE
Feature request -  [Subtitle voiceover]

### ISSUE THREAD

**Issue by bropines** [snakers4/silero-models#226]
## 🚀 Feature
A simplified system that simply asks for srt and other types of sub files, and voices them by timings.

## Pitch

I wanted it to be as simple as Whisper. 
That is: `silero-tts --file oleg.srt --sample-rate 48000 --model olesya --lang ru`

## Alternatives

There are no alternatives on the Internet that can do this in Russian and relatively easy

**Comment by dmki** [comment]
If you are on Windows, you could use SoundWorks to generate Silero voiceover for SRT. I don't think it should be part of the engine itself to do such custom niche tasks. Third party apps that consume Silero might handle it better.

In fact, you could do it yourself.

**Comment by bropines** [comment]
When this question was asked, I did not understand anything in the documentation. Now, with experience, we should try again.

**Comment by snakers4** [comment]
The requested feature should not be a part of tts engine.
